VirtoCommerce.CacheModule.Data/VirtoCommerce.CacheModule.Data.csproj

```diff
@@ -105,7 +105,9 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="App.config" />
-    <None Include="packages.config" />
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
```

Comment:
Do we need this line?

Reply:
Looks like line adding is a known issue: https://developercommunity.visualstudio.com/content/problem/204355/subtype-designer-is-added-unnecessarily-to-xml-bas.html and should be fixed in Visual Studio 2019 Preview 2.